release-3.7.0-announcement.md

```diff
@@ -0,0 +1,31 @@
+We're excited to announce the availability of Jamulus 3.7.0!
+
+This release re-organizes Server lists to make room for more servers:
+- Default has been renamed to Any Genre 1.
+- Any Genre has been renamed to Any Genre 2.
+- Any Genre 3 has been added.
+- Choral/Barbershop has been added.
+- Classical/Folk/Choral has been renamed to Classical/Folk.
+
+Besides that, tens of usability improvements, enhancements, bugfixes and optimizations have been integrated.
+Please find all the details in the [Changelog](https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0).
+
+Downloads:
+- [Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_win.exe)
+- [Mac](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_mac.dmg)
+- [Debian/Ubuntu (amd64)](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_ubuntu_amd64.deb), alternative: [headless version](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_headless_3.7.0_ubuntu_amd64.deb)
+- [Source code](https://github.com/jamulussoftware/jamulus/archive/r3_7_0.tar.gz)
+
+This is the first release after the founder, developer and project lead of Jamulus, Volker Fischer (@corrados), [decided to leave the project](https://github.com/jamulussoftware/jamulus/discussions/928).
+We want to use this opportunity to thank Volker again. Creating Jamulus and pushing it to the point where it is today was and is a huge accomplishment! Thanks, Volker!
+
+Starting with this release, the project will be lead by the Main Developers Team.
```

Comment:
We should put a link to https://github.com/orgs/jamulussoftware/teams/maindevelopers on the "Main Developers Team", then every users can know precisely who are the people under the "Main Developers Team".

Reply:
> then every users can know precisely who are the people under the "Main Developers Team".

I'd love to, but I'm not sure that this link will help as it is only accessible to logged-in users (possibly only users which are part of that particular team or at least the organization).

Any other ideas?

Reply:
We could manually maintain a list of people on the website page (eg jamulus.io/team)?

Reply:
Yes, I would certainly welcome that. Is this something doable within a day or would this rather be something for the next release?

Reply:
I think we'd have to wait for the next release as we'd need to make sure everyone was happy with the idea (and for them to add any bio stuff etc.)

Reply:
https://github.com/jamulussoftware/jamuluswebsite/issues/355 has been opened by @gilgongo  meanwhile.